src/Microsoft.Identity.Web/AuthorityHelpers.cs

```diff
@@ -10,8 +10,8 @@ internal static class AuthorityHelpers
     {
         internal static string BuildAuthority(MicrosoftIdentityOptions options)
         {
-            var baseUri = new Uri(options.Instance);
-            var pathBase = baseUri.PathAndQuery.TrimEnd('/');
+            Uri baseUri = new Uri(options.Instance);
```

Comment:
By default, `var` becomes the nullable reference type (`Uri?` in this case). So in some cases I used explicit type instead.